src/Microsoft.DotNet.Wpf/src/PresentationFramework/MS/Internal/documents/TextBoxView.cs

```diff
@@ -3263,11 +3263,11 @@ internal int Offset
         private EventHandler UpdatedEvent;
 
         // Max time slice to run FullMeasureTick.
-        private const uint _maxMeasureTimeMs = 200;
+        private static TimeSpan _maxMeasureTime = TimeSpan.FromMilliseconds(200);
```

Comment:
Could you make the readonly field as `static readonly` key word

Reply:
Done